src/App.js

```diff
@@ -14,11 +17,49 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+const goodsItems = goodsFromServer.map((item, index) => ({
+  id: index + 1,
+  name: item,
+}));
+
+class App extends Component {
+  state = {
+    selectedGoods: [],
+  };
+
+  clickHandler = (item) => {
+    this.setState((changeState) => {
+      if (changeState.selectedGoods.includes(item)) {
+        const goodsWord = changeState.selectedGoods
+          .filter(good => good !== item);
+
+        return { selectedGoods: goodsWord };
+      }
+
+      const selectedWords = [...changeState.selectedGoods, item];
+
+      return { selectedGoods: selectedWords };
```

Comment:
ну если на то уже пошло то тогда
```suggestion
      const selectedGoods = [...changeState.selectedGoods, item];

      return { selectedGoods };
```